.travis.yml

```diff
@@ -2,4 +2,10 @@ language: node_js
 node_js:
   - "lts/*"
   - "node"
-script: npm run ci
\ No newline at end of file
+script: npm run ci
+deploy:
+  provider: npm
+  email: "abenojaalex@gmail.com"
```

Comment:
You okay having your email address in here? Do we want to create a special deployment email address?

Reply:
Honestly I'm not too worried about it. My email is publicly available in the commits anyway.